src/common/containers/build-details.js

```diff
@@ -60,9 +61,23 @@ class BuildDetails extends Component {
         }
         { build &&
           <div>
-            <BuildRow repository={repository} {...build} />
-            <h3>Build log:</h3>
-            <BuildLog logUrl={build.buildLogUrl} />
+            <Table>
+              <Head>
+                <Row>
+                  <Header col="20">Number</Header>
+                  <Header col="20">Architecture</Header>
+                  <Header col="20">Duration</Header>
+                  <Header col="40">Result</Header>
+                </Row>
+              </Head>
+              <Body>
+                <BuildRow repository={repository} {...build} />
+              </Body>
+            </Table>
+            <h3>Build log</h3>
```

Comment:
Should be a `HeadingThree` to keep consistent with vanilla headings.

Also right now it seems that `styles.strip` give build log quite a large top/bottom margin, so maybe it would be good to move heading inside of the strip, so it's closer to build log.